cmd/lakectl/cmd/fs.go

```diff
@@ -166,7 +165,7 @@ func uploadObject(ctx context.Context, client api.ClientWithResponsesInterface,
 		defer func() {
 			_ = pw.Close()
 		}()
-		filename := path.Base(filePath)
+		filename := filepath.Base(filePath)
```

Comment:
nit; It is confusing that both the library and variable names called filePath. maybe rename the variable to path?

Reply:
@talSofer - I checked and the filepath here is actually the destination path, so decided to go with objectPath after all